pkg/driver_atwinc15x0/Makefile

```
@@ -0,0 +1,20 @@
+PKG_NAME=driver_atwinc15x0
+PKG_URL=https://github.com/arduino-libraries/WiFi101
+PKG_VERSION=3301d03f82e53f60e07434a9f07ca677d992d121
+PKG_LICENSE=LGPL-2.1
+
+include $(RIOTBASE)/pkg/pkg.mk
+
+CFLAGS += -Wno-discarded-qualifiers
+CFLAGS += -Wno-empty-body
+CFLAGS += -Wno-old-style-definition
+CFLAGS += -Wno-unused-parameter
+CFLAGS += -Wno-incompatible-pointer-types-discards-qualifiers
+
+CFLAGS += -DETH_MODE
+CFLAGS += -I$(PKG_BUILDDIR)/src
+
+all:
+	"$(MAKE)" MODULE=driver_atwinc15x0 -C $(PKG_BUILDDIR)/src/driver/source -f $(RIOTBASE)/Makefile.base
+	"$(MAKE)" MODULE=driver_atwinc15x0_common -C $(PKG_BUILDDIR)/src/common/source -f $(RIOTBASE)/Makefile.base
+	"$(MAKE)" MODULE=driver_atwinc15x0_spi_flash -C $(PKG_BUILDDIR)/src/spi_flash/source -f $(RIOTBASE)/Makefile.base
```

#14289 was merged so this Makefile must be adapted once again:
```suggestion
	"$(MAKE)" -C $(PKG_SOURCE_DIR)/src/driver/source -f $(RIOTBASE)/Makefile.base MODULE=driver_atwinc15x0
	"$(MAKE)" -C $(PKG_SOURCE_DIR)/src/common/source -f $(RIOTBASE)/Makefile.base MODULE=driver_atwinc15x0_common
	"$(MAKE)" -C $(PKG_SOURCE_DIR)/src/spi_flash/source -f $(RIOTBASE)/Makefile.base MODULE=driver_atwinc15x0_spi_flash
```
(I also propose to move the `MODULE=xxx` at the end of the line, it's purely a cosmetic change regarding the build output)

---

OK

---

May I squash these changes directly to avoid an extra loop in the long, long CI compilation queue?

---

> May I squash these changes directly to avoid an extra loop in the long, long CI compilation queue?

Sure, let's save some CI time.